tool/travis.sh

```diff
@@ -40,4 +40,11 @@ if [ "$REPO_TOKEN" ]; then
     --retry 2 \
     --exclude-test-files \
     test/all.dart
-fi
\ No newline at end of file
+fi
+
+# Globally install grinder.
+pub global activate grinder
+export PATH=~/.pub-cache/bin:$PATH
+
+# Run dartdoc.
+grind test
```

Comment:
I generally use `pub global run grind` here, instead of just `grind`, as it's a bit more robust.